docs/source/architecture/02_validated_architecture.md

```diff
@@ -0,0 +1,51 @@
+# Validated Architecture
+
+This layout involves a lot more resources than the lightweight one.
+The schema bellow will help understanding the networking and communications
+
+## Layout overview
+
+The following schema will help understanding the validated architecture layout:
+
+```
+._____________________________________.
+|          Hypervisor                 |
+|                                     |
+|  [controller-0]                     |
+|      | |                            |
+|      | '---------------------.      |
+|      '-----------------.     |      |
+|                  osp_trunk   |      |
+|  [ ocp-0/1/2 ]         |   ocpbm    |
+|      | |               |     |      |
+|      | '---------------|     |------|--{Internet, external resources}
+|      '-----------------|-----|      |
+|                        |     |      |
+|  [ compute-0/1/2 ]     |     |      |
+|     |  |   |           |     |      |
+|     |  |   |           |     |      |
+|     |  |   '-----------'     |      |
+|     |  '---------------------'      |
+|     '------ ocppr                   |
```

Comment:
`ocppr` bridge is connected with the OCP nodes as well

Reply:
woops, actualyl it's only on the ocp nodes... good catch - need to move things around